src/css/color.zig

```diff
@@ -213,24 +268,33 @@ const NAMED_COLORS = .{
 };
 
 test "Color.parse()" {
-    try expectParse(Color, "#000000", .{ .r = 0, .g = 0, .b = 0, .a = 0xFF });
-    try expectParse(Color, "#ff0000", .{ .r = 0xFF, .g = 0, .b = 0, .a = 0xFF });
-    try expectParse(Color, "#00ff00", .{ .r = 0, .g = 0xFF, .b = 0, .a = 0xFF });
-    try expectParse(Color, "#0000ff", .{ .r = 0, .g = 0, .b = 0xFF, .a = 0xFF });
+    try expectParse(Color, "#000000", .{ .r = 0, .g = 0, .b = 0, .a = 255 });
```

Comment:
either keep the original `0xFF` or change other components as well, this looks weird.